resources/build/trigger-definitions.inc.sh

```diff
@@ -61,4 +67,13 @@ bc_beta_web=(Keymanweb_Build)
 
 vcs_beta=HttpsGithubComKeymanappKeyman
 
-# TODO: Stable Build Configurations
+# Stable 14.0 Build Configurations
+
+bc_stable_14_0_android=(KeymanAndroid_Build)
+bc_stable_14_0_ios=(Keyman_iOS_Master)
+bc_stable_14_0_linux=(KeymanLinux_Master pipeline-keyman-packaging_Jenkins)
+bc_stable_14_0_mac=(KeymanMac_Master)
```

Comment:
Too bad some of the configs have `*Master` in the name for builds that aren't always `master` branch.

Reply:
Yeah, see comments on line 22-23:

```shell
# Available build configurations and VCS identifiers; identifiers are somewhat inconsistent due
# to legacy updates and changes.
```